README.md

```diff
@@ -2,39 +2,49 @@
 [uri_license]: https://www.mozilla.org/en-US/MPL/2.0
 [uri_license_image]: https://img.shields.io/badge/MPL-2.0-blue.svg
 
-![github-light-mode](https://github.com/penpot/penpot/assets/5446186/e5e725fb-a925-43aa-904e-68658c2ef1b5)
-
+<picture>
```

Comment:
This could be done using simple Github markdown according to this: https://dev.to/arnavkr/using-darklight-mode-specific-images-in-github-markdown-3iln

```
![Penpot Logo](https://penpot.app/images/readme/github-dark-mode.png#gh-dark-mode-only)
![Penpot Logo](https://penpot.app/images/readme/github-light-mode.png#gh-light-mode-only)
```

Reply:
this method is officially deprecated:

https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax#specifying-the-theme-an-image-is-shown-to